### PR TITLE
Update + enable (fast!) llvmpipe software renderer

### LINKED ISSUE #41: Needs update, 22.3.3 is out
See https://docs.mesa3d.org/relnotes/22.3.3.html

### PR DESCRIPTION
**Pull request by rggjan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Closes #41 

Update to the latest version.

Also, enable the llvmpipe renderer, which allows for much faster (10x or more) software rendering within mesa, as llvmpipe is optimized + can run multi-threaded.

### CODE REVIEW

**Comment by rggjan** [comment]
@conda-forge-admin, please rerender

**Comment by rggjan** [comment]
@conda-forge-admin please rerender

**Comment by rggjan** [comment]
@conda-forge-admin, please rerender

**Comment by rggjan** [comment]
@conda-forge-admin, please rerender

**Comment by rggjan** [comment]
Can anyone help with getting the osx version to run? Not sure what's the problem there...

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# New versions need clock_gettime
-# which is supported in 10.12 and later
+# New versions need open_memstream
+# which is supported in 10.13 and later
 macos_min_version:             # [osx and x86_64]
-  - 10.12                      # [osx and x86_64]
+  - 10.13                      # [osx and x86_64]
```
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml:4]: did you try maybe increasing this even more?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -57,6 +58,7 @@ requirements:
     - xorg-libxfixes
     - xorg-glproto
     - xorg-libxdamage
+    - llvm
```
Comment (hmaarrfk) [inline recipe/meta.yaml:61]: do you need llvm here? llvm should export its own dependencies

Comment (rggjan) [reply recipe/meta.yaml:61]: Oh I wasn't aware. Removed.

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (rggjan) [reply recipe/conda_build_config.yaml]: Didn't work unfortunately. I'll try disabling the llvm stuff on OSX (where it's not that relevant anyway as OpenGL can be used)

**Comment by rggjan** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
Maybe you want to downgrade clang

https://gitlab.freedesktop.org/mesa/mesa/-/issues/7804

See the changes i made to the conda-build-config.yaml file in
https://github.com/conda-forge/libnetcdf-feedstock/issues/138

**Comment by rggjan** [comment]
@conda-forge-admin, please rerender

**Comment by rggjan** [comment]
OK, seems like this finally worked :)

**Comment by hmaarrfk** [comment]
What did you do to fix osx?

**Comment by rggjan** [comment]
> What did you do to fix osx?

I disabled llvm for osx. The update to 10.13 was still needed to make it compile with the newest version of mesalib.

**Comment by rggjan** [comment]
Anything missing to merge this?

**Pull request merged**